bin/pc2reset

```diff
@@ -5,18 +5,18 @@
 # Author : pc2@ecs.csus.edu
 #
 
-. `dirname $0`/pc2env
+. "$(dirname "$0")/pc2env"
 
 # clients have a logs directory or servers have a profiles directory
-if [ -d logs -o -d profiles ] ; then
-    `dirname $0`/pc2zip
-    if [ -d archive ]; then
-	rm -fr logs packets reports execute* profiles profiles.properties results
-    else
-	echo Could not find archive directory, was pc2zip successful?
-    fi
+if [ -d logs ] || [ -d profiles ]; then
```

Comment:
This change is not related to security, and will require additional testing, there are a number
of rewrites of extisting well tested code.

Reply:
I'm sorry, I think you misunderstood my usage of the word "safe" in the description. None of these changes should directly affect the security of the scripts. They simply improve edge-case handling and use more modern shell features to ensure higher success rates across a wide variety of machines.

Reply:
This specific change from the `[ cond -o cond ]` syntax to `[ cond ] || [ cond ]` syntax should make no measurable difference in a wide majority of cases, however, as these scripts run in `/bin/sh`, which is often provided by different shells (`bash` in some systems, `dash` in others such as mine, etc), it is generally advised to stick to POSIX compatible syntax (such as `||`) rather than shell extensions which may not be as portable between machines (such as `-o`)

Reply:
From the [POSIX definition of test (`[]`)](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html)
```
expression1 -o  expression2
[OB XSI]
True if either expression1 or expression2 is true; otherwise, false. The -o binary primary is left associative.
```

In which, the `[OB XSI]` classification is described as:

```
The functionality described may be removed in a future version of this volume of POSIX.1-2017. Strictly Conforming POSIX Applications and Strictly Conforming XSI Applications shall not use obsolescent features.
```

As thus, for future compatibility and portability in all POSIX shells (which `/bin/sh` is supposed to be, regardless of it's implementation), using `[ cond ] || [ cond ]` is the recommended syntax over the test operator `-o`.